virtualmachine/vsphere/util.go

```diff
@@ -987,6 +991,30 @@ func WaitForGuestStatus(vm *VM, vmMo *mo.VirtualMachine, status int) error {
 	return err
 }
 
+// waitForRunningTools: wait for vmware tools status to be running
+func waitForRunningTools(vm *VM, vmMo *mo.VirtualMachine) error {
```

Comment:
Waiting for Tools status may not be sufficient. Let's wait for Guest status only.

Reply:
Remove check for vmware tools, added timeout for guest status check